components/shooter.py

```diff
@@ -121,9 +121,15 @@ def __init__(self) -> None:
         self.inclinator_controller.setTolerance(ShooterComponent.INCLINATOR_TOLERANCE)
         SmartDashboard.putData(self.inclinator_controller)
 
+        self.range = 0
+
     def on_enable(self) -> None:
         self.inclinator_controller.reset()
 
+    @feedback
+    def current_range(self):
+        return self.range
+
```

Comment:
nit: tunable can be used instead of feedback to send a single variable
```suggestion
    range = tunable(0.0)

    def on_enable(self) -> None:
        self.inclinator_controller.reset()

```